api/Trace/Links.php

```diff
@@ -11,9 +11,7 @@ public function getIterator(): LinksIterator;
 
     /**
      * Adding a link should not invalidate nor change any existing iterators.
-     * @param SpanContext $context
-     * @param Attributes|null $attributes
```

Comment:
Should we add `@param Link` as a string here?

Reply:
Would there be any benefit in doing that given it already has a `Link` type hint?